Cargo.toml

```diff
@@ -1,9 +1,23 @@
 [workspace]
 
 members = [
+  # Tooling
   "cli",
-  "runtime",
-  "guests/function"
+  "development_server",
+  "wasmtime_components",
+
+  # Services
+  "services/function",
+
+  # Guests
+  "guests/function",
```

Comment:
For my understanding: what's the difference between a `service` and a `guest`

Reply:
Good question 🤔 . So, code within these packages should provide the server side implementations needed to run guests. Maybe an alternative like `hosts` or `tenants` makes more sense?

Reply:
Yeah that would certainly make their relation more obvious! Great suggestion and I'll re-read the code while I keep this distinction in mind

Reply:
Mind if I rename these in a follow up PR? Want to get this merged in while its in a working state.

Reply:
Sure! All good... would be helpful for me too if this was merged so I can apply the changes to the xtasks PR